validator/sawtooth_validator/networking/handlers.py

```diff
@@ -15,8 +15,12 @@
 import logging
 import os
 import time
+
+from urllib3.util import url
```

Comment:
This needs to be an explicit dependency.  It might be a transient dependency, which could be the reason it works, but that may only be due to a core dependency requiring it currently.

Reply:
Hi - can you tell me what to change? I do not know what I need to do at this point.

Reply:
Would it be possible to make this change using the built-in urllib?